### ISSUE TITLE
Uneven balancing

### ISSUE THREAD

**Issue by mooop12** [cwmonkey/reactor-knockoff#4]
It's hard to explain so I'll just show a screenshot:
http://i.imgur.com/9jlV8U3.png
This is stable at the moment, but if I were to add a few more batteries (or plates), the left heat exchanger would blow up first, followed by the top, bottom and then right (always in this order) after adding more batteries. Ratio of heat output of outlets to heat dissipation of vents stays the same (1 outlet per 16 vents).

Just to be clear, those heat vents are supposed to be enough to dissipate all the heat coming from the outlet, according to the numbers in the tooltip, so all the heat should be zero according to my understanding.

**Comment by cwmonkey** [comment]
Yeah, this algorithm has always been a pain for me. Thanks for the details on causing the issue, should make it easier for me to debug.
